Src/IronPython.Modules/fcntl.cs

```diff
@@ -1,8 +0,0 @@
-
```

Comment:
It wasn't included in the project before (and didn't compile). Including it also causes failures in stuff that was working before (e.g. tempfile).